desc/compute/utils.py

```diff
@@ -12,7 +12,21 @@
 from .data_index import data_index
 
 # defines the order in which objective arguments get concatenated into the state vector
-arg_order = ("R_lmn", "Z_lmn", "L_lmn", "p_l", "i_l", "c_l", "Psi", "Rb_lmn", "Zb_lmn")
+arg_order = (
+    "R_lmn",
+    "Z_lmn",
+    "L_lmn",
+    "p_l",
+    "i_l",
+    "c_l",
+    "Psi",
+    "Te_l",
+    "ne_l",
+    "Ti_l",
+    "Zeff_l",
```

Comment:
My idea was to preserve the existing order where possible, the exception being the boundary terms because in some places we assume those are the last elements.

Reply:
OK. At some point we should probably fix those parts of the code that assume the boundary terms are the final parameters (perturbations, anywhere else?) so that this arg order can be completely arbitrary.